javascript/javascript1/week1/Ez-Namey.js

```diff
@@ -0,0 +1,7 @@
+const firstWords = ['Mercury', 'Venus', 'Earth', 'Mars', 'Jupiter', 'Saturn', 'Uranus', 'Neptune', 'Stars', 'Sun'];
+const secondWords = ['Dog', 'Puppy','Turtle', 'Rabbit', 'Parrot', 'Cat', 'Kitten', 'Goldfish', 'Mouse', 'Hamster'];
+const FirstRandomNumber = Math.floor(Math.random() * 10) + 0;
+const SecondRandomNumber = Math.floor(Math.random() * 10) + 0;
```

Comment:
These don't need to be capitalized. 
You should stick with one naming strategy for variables.

Reply:
I know it's in the homework but the `+ 0` doesn't do anything here :)